Loop/Managers/LoopDataManager.swift

```diff
@@ -39,6 +39,14 @@ final class LoopDataManager {
             notify(forChange: .preferences)
         }
     }
+    
+    var batteryIsLithium: Bool {
```

Comment:
Yes with the above UI recommendation to select a battery chemistry type, I think that batteryIsLithium will be unused and can be removed.